uxarray/grid.py

```diff
@@ -211,7 +214,8 @@ def __from_vert__(self, dataset):
             # Update indices accordingly
             for i, idx in enumerate(false_indices):
                 # TODO: setting the INT_FILL_VALUE as np.iinfo(INT_DTYPE).max will cause "OverflowError: Python int too large to convert to C long" here
-                indices[indices == idx] = -1 # TODO: Replace -1 with  INT_FILL_VALUE later
+                indices[indices ==
```

Comment:
@philipc2 
If I tried 
```python
indices[indices == idx] = INT_FILL_VALUE
```

It will throw
```python
Victoria Chan
            # Update indices accordingly
            for i, idx in enumerate(false_indices):
>               indices[indices == idx] = indices.dtype.type(INT_FILL_VALUE)
E               OverflowError: Python int too large to convert to C long
```

So I just use `-1` as the fill_value for now. Can you help me fix this so that we can use `INT_FILL_VALUE` here? Thanks

Reply:
Currently, I temperately solved it by converting `-1` back to `INT_FILL_VALUE` later
in the connectivity array

```python
        connectivity = indices.reshape(dataset.shape[:-1])
        connectivity = np.where(connectivity == -1, INT_FILL_VALUE,
                                connectivity)
```

But I wonder if there's a better solution for it

Reply:
You need to ensure that your arrays are all of type `INT_DTYPE`, in this case `indicies`.